org.openhab.binding.zigbee/src/main/resources/discovery.txt

```diff
@@ -11,3 +11,7 @@ xiaomi_lumisensor-switchaq2,modelId=lumi.sensor_switch.aq2
 xiaomi_lumisensorwaterleak,modelId=lumi.sensor_wleak.aq1
 innr-rc-110,vendor=innr,modelId=RC 110
 osram-switch-4x-eu,vendor=OSRAM,modelId=Switch 4x EU-LIGHTIFY
+tuya_ts0041,vendor=_TZ3000_tk3s5tyg,modelId=TS0041
```

Comment:
We could remove the vendor property or change this to a regex match, e.g. _TZ3000_\w{8} .

Reply:
removed the vendor instead. modelId should be sufficient.